### ISSUE TITLE
AttributeErrror

### ISSUE THREAD

**Issue by vexillia** [woodruffw/autopost#38]
I'm try to post from two feeds to the same account:

```
autopost --config-file "/home/vexillia/.autopost.toml" atom https://blog.vexillia.me.uk/feeds/posts/default
autopost --config-file "/home/vexillia/.autopost.toml" atom https://blog.vexillia.me.uk/feeds/comments/default
```
The first works but the second gives this error:

`AttributeError: object has no attribute 'tags'`

I have  a third feed which gives the same error.

**Comment by woodruffw** [comment]
Can you please provide the full backtrace? It's hard to know where that `AttributeError` is coming from without it.

**Comment by vexillia** [comment]
As requested: 

[backtrace.txt](https://github.com/user-attachments/files/17692222/backtrace.txt)

**Comment by woodruffw** [comment]
Thanks. Yeah, it looks like that second feed has no `tags` in its entries, which `autopost` expects as part of giving each post categories/hashtags/etc.

I'll relax that requirement, but in the mean time you could also add some tags to get it to work as-is.

**Comment by woodruffw** [comment]
Fixed in v0.0.8, which should be live on PyPI shortly.

**Comment by vexillia** [comment]
Cool and thanks again.
